tools/analysis_tools/test_robustness.py

```diff
@@ -207,8 +207,18 @@ def main():
         init_dist(args.launcher, **cfg.dist_params)
 
     # set random seeds
-    if args.seed is not None:
-        set_random_seed(args.seed)
+    if hasattr(cfg, "seed") and args.seed is not None:
+        seed = args.seed
+        print(f'Argument seed passed both in config file and as an argument. \
+            Please avoid setting the seed in the config file if also passing it as an argument. \
+            Using argument random seed {args.seed}')
+    elif args.seed is not None:
+        seed=args.seed
+    elif hasattr(cfg, "seed"):
+        seed=cfg.seed
+    else:
+        seed=None   
+    set_random_seed(seed)
```

Comment:
miss `seed = init_random_seed(seed)`

Reply:
added, thanks

Reply:
@hhaAndroid @RangiLyu I fixed all the `flake8` issues, imported `init_random_seed` in `tools/analysis_tools/test_robustness.py ` (it was missing). Can you tell me what else is needed to close this PR? BTW, it appears that the linter checks are not repeated after I pushed the new commits, and thus they still show as failing. Should I close this PR and open a new one?

Reply:
No need to close, just use this PR directly.